### ISSUE TITLE
Notary returned 401

### ISSUE THREAD

**Issue by leeadh** [theupdateframework/notary#1367]
Dear all, 

I tried to automate this via jenkins . Basically, I exported my passphrase + exported the notary into the /etc/profile and ran jenkins. My jenkins is reading from a github repo where my docker file +jenkins file + code resides. 

export DOCKER_CONTENT_TRUST=1
export DOCKER_CONTENT_TRUST_ROOT_PASSPHRASE="XXX!"
export DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE="XXX!"
export DOCKER_CONTENT_TRUST_SERVER=https://notary.docker.io

Signing and pushing trust metadata
you are not authorized to perform this operation: server returned 401.

Any help please? Do we really need to create a notary service because looking at the standard documentation of content trust(https://docs.docker.com/engine/security/trust/content_trust/#content-trust-operations-and-keys) , u can export docker content trust =1 and i will be able to push the signed image to docker hub. I tried this on cmd line and it works hence I assume its using the default notary. therefore why do we need to create one notary?

**Comment by cyli** [comment]
@leeadh Could you give more details?  Are you pushing data to docker hub?  If so, if you are authorized to write to the repository on docker hub, then the above environment variables should work.

**Comment by leeadh** [comment]
Hi cyli yes I am trying to push image to docker hub. Is there url of what I wrote down correct?

**Comment by cyli** [comment]
@leeadh Yes, those environment variables are correct.  So are you trying to do `docker push <your hub username>/<image>:tag` or something like that?  If so, and the image name is not sensitive, can you please run `docker -D push <hub username>/<image>:tag` instead and paste all the output?  That just runs it in debug mode so we get some more info.

**Comment by leeadh** [comment]
Hello @cyli 

I am able to push in command line 

![image](https://user-images.githubusercontent.com/15122358/41809538-f4407366-7721-11e8-9af7-d14d6a2b41a0.png)

**Comment by cyli** [comment]
Just double-checking - are you using the same user/same image name on both jenkins and on your desktop?  Does it repeatedly fail in jenkins?

**Comment by leeadh** [comment]
HI @cyli  any thing?
